pkg/log/messages/info_messages.go

```diff
@@ -29,3 +29,4 @@ const CSPFK014I string = "CSPFK014I Authenticator setting %s provided by %s"
 const CSPFK015I string = "CSPFK015I DAP/Conjur Secrets pushed to shared volume successfully"
 const CSPFK016I string = "CSPFK016I There are no secrets to be retrieved from Conjur"
 const CSPFK017I string = "CSPFK017I Creating default file name for secret group '%s'"
+const CSPFK018I string = "CSPFK018I No change in secret files, no secret files written"
```

Comment:
I believe that this should be singular "secret file" (for both instances), i.e.
`CSPFK018I No change in secret file, no secret file written`